extensions/ql-vscode/.storybook/preview.js

```diff
@@ -0,0 +1,38 @@
+import { themes } from '@storybook/theming';
+import { action } from '@storybook/addon-actions';
+
+// Allow all stories/components to use Codicons
+import '@vscode/codicons/dist/codicon.css';
+
+import '../src/stories/vscode-theme.css';
+
+export const parameters = {
```

Comment:
Were these just generated? Would be good to add some context about what they are, maybe a link to the docs?

Reply:
Most of these are from the template, do you think it would be enough to link to [this documentation page](https://storybook.js.org/docs/react/configure/overview#configure-story-rendering)?